### ISSUE TITLE
pleas add logseqpagetitle

### ISSUE THREAD

**Issue by mrVanDalo** [ralphbean/bugwarrior#1064]
I like the Logseq service a lot, but I would be nice to also put the page title in the description. e.g.: `{{logseqpagetitle}}`

![image](https://github.com/user-attachments/assets/348e2d88-52ea-4eb5-835d-46be15663765)

**Comment by ryneeverett** [comment]
It seems like it would make sense to add logseqpagetitle as a UDA, which would be sufficient to enable you to add it to the description via [field templating](https://bugwarrior.readthedocs.io/en/latest/common_configuration.html#field-templates).

I suspect adding it to the default description would not be great for most users as it would make the description even longer and somewhat misaligned with the default descriptions of other services.

**Comment by mrVanDalo** [comment]
I agree, adding it as UDA is enough. It would make the description confusing if it would be in there by default.

**Comment by twaugh** [comment]
Might there also be a case for using the page title (rather than the graph name) as the project associated with the task? I'd find that useful.
